apps/frontend/src/components/WorkoutMenu/WorkoutEditor.tsx

```diff
@@ -352,6 +378,59 @@ export const WorkoutEditor = ({
           </Tr>
         </Tfoot>
       </Table>
+      <AlertDialog
+        isOpen={showDeleted}
+        leastDestructiveRef={cancelRef}
+        onClose={() => setShowDeleted(false)}
+      >
+        <AlertDialogOverlay>
+          <AlertDialogContent>
+            <AlertDialogHeader fontSize="lg" fontWeight="bold">
+              Delete workout
+            </AlertDialogHeader>
+
+            <AlertDialogBody>
+              This will delete the workout. Are you sure?
+            </AlertDialogBody>
+
+            <AlertDialogFooter>
+              <HStack>
+                <Button
+                  colorScheme="red"
+                  onClick={() => {
+                    setShowDeleted(false);
+                    deleteWorkout();
+                  }}
+                  ml={3}
+                >
+                  Yes, delete the workout
+                </Button>
+                <Button ref={cancelRef} onClick={() => setShowDeleted(false)}>
+                  Cancel
+                </Button>
+              </HStack>
+            </AlertDialogFooter>
+          </AlertDialogContent>
+        </AlertDialogOverlay>
+      </AlertDialog>
```

Comment:
A bit boilerplaty. 
Lot of same code as the "discard workout changes"-dialog.
Could be put into a reusable component i guess